pkg/route/validation/validation.go

```diff
@@ -342,6 +382,56 @@ func validateWildcardPolicy(host string, policy routev1.WildcardPolicyType, fldP
 	return nil
 }
 
+var (
+	notAllowedHTTPHeaders   = []string{"strict-transport-security", "proxy", "cookie", "set-cookie"}
+	notAllowedHTTPHeaderSet = sets.NewString(notAllowedHTTPHeaders...)
+)
+
+func validateHeaders(path *field.Path, headerValues []routev1.RouteHTTPHeader) *field.Error {
+	unique := make(map[string]bool, len(headerValues))
+	us := make([]string, len(unique))
+	for key, value := range headerValues {
+		//for _, elem := range s {
```

Comment:
Remove the commented code.

Reply:
sorry! thanks for pointing it!

Reply:
https://github.com/openshift/library-go/commit/376ae6a3fee9be3ed62f1265c81cb8f535e8a2b5#diff-bfc6ba95cbc94986901ce1d0e3b74ee966499eab85a96e1d1ff763d2d74a7eeeR396